pandas/io/formats/style_render.py

```diff
@@ -160,7 +178,7 @@ def _compute(self):
             r = func(self)(*args, **kwargs)
         return r
 
-    def _translate(self):
+    def _translate(self, blank="&nbsp;"):
```

Comment:
maybe should rename this to _translate_html

Reply:
at the moment for HTML and LaTeX `_translate` is necessary: its a base translate common to both. For LaTeX there is the additional `_translate_latex` which is an addon to the base. 

In this knowledge do you still want the name change (I didn't want to replicate code in two different translate methods, and I didn't want to pollute the base translate with LaTeX elements)